plugins/TelescopeControl/src/Lx200/TelescopeClientDirectLx200.hpp

```diff
@@ -89,7 +90,7 @@ class TelescopeClientDirectLx200 : public TelescopeClient, public Server
 		return interpolatedPosition.isKnown();
 	}
 
-	Equinox equinox;
```

Comment:
schemmatic of the vectors is the same

Reply:
schematic of which vectors is the same? Please explain your concerns.

Reply:
the schematic of v and omega, on the mathematical form and in this connected planes interpolatedPosition as V which found on i module and sees iPdP coding measurements counted and omega as space after return base.